lib/services/env file

```diff
@@ -0,0 +1,6 @@
+//use this as a reference to the original .env file located at the '..ISN/DEVELOPMENT/' folder
+//this .env file is already saved in my project's root file on the same level as pubspec.yaml 
+
+
+SUPABASE_URL=https://vfoicylmtasvhrnmdnki.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InZmb2ljeWxtdGFzdmhybm1kbmtpIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDY5MDM0ODUsImV4cCI6MjA2MjQ3OTQ4NX0.flMKhV42hhPFXQQjjM78L45Iekj34YfZKbntz__RiLg
```

Comment:
The Supabase URL and AuthKey should not be pushed to the GitHub repository. They are secrets that should only be stored on the client side or the hosting server once the application is deployed, to prevent leakage of sensitive data. Please hide the Supabase URL and AuthKey in a .env file or similar package and ensure they are not pushed to GitHub.